### ISSUE TITLE
Hyperpixel4 (Square) showing black screen with RPi 5 [Bookworm (64bit)]

### ISSUE THREAD

**Issue by n-ubo** [pimoroni/hyperpixel4#214]
trying the approaches listed here in issue 208:
https://github.com/pimoroni/hyperpixel4/issues/208
and
https://github.com/pimoroni/hyperpixel4/issues/177#issuecomment-1837186505

i still did not manage to get the screen to work with the Pi 5 in combination with the hyperpixel4 **square**. 

what did i do?
i manually installed the rpi4 branch, and changed the install.sh script's config to the new config.txt location that the rpi5 has inside `/boot/firmware`. rebooted, and now the screen shows black. however, when i connect a screen to one of the micro hdmi ports, and touch the hyperpixel4's touch screen, the touch seems to move the cursor on the other monitor just fine.

i have had a similar issue with the rpi3b before and managed to fix it by commenting `dtoverlay=vc4-kms-v3d` in the config.txt. However, trying to do the same in the rpi5's `/boot/firmware/config.txt` it gets stuck on reboot.
![image](https://github.com/pimoroni/hyperpixel4/assets/72999487/ae3706e2-e519-4112-9a62-751958623e92)
note that the system consists of a rpi 5, hyperpixel4 square and a PIjuice hat, and that this combination worked for the rpi 3 and 4.

**Comment by JacobHJensen** [comment]
Hi Nuboctane, did you ever get this to work? :-)

**Comment by n-ubo** [comment]
> Hi Nuboctane, did you ever get this to work? :-)

nope, went with another pi 3 i still had, this was meant for a handheld and saved me power consumption in the end, the question still stands.
